packages/gradle/src/plugin/utils/gradle-plugin-options.ts

```diff
@@ -1,6 +1,7 @@
 export interface GradlePluginOptions {
   testTargetName?: string;
   ciTestTargetName?: string;
+  customGradleInstallation?: string;
```

Comment:
```suggestion
  gradleExecutableDirectory?: string;
```

I think this works better?